src/AVFoundation/AVAudioRecorder.cs

```diff
@@ -104,6 +104,12 @@ public AVAudioRecorder (NSUrl url, NSDictionary settings, NSError outError)
 			Handle = InitWithUrl (url, settings.Dictionary, out error);
 		}	
 
+		[iOS (10,0), Mac (10,12)]
```

Comment:
did you meant this to be public ?

If not (and it's just for `Create`) then there's no need for the attributes. They are only needed to public/protected members or when they are `[Export]` or `[DllImport]`